.travis.yml

```diff
@@ -7,6 +7,7 @@ python:
 
 install:
 - pip install -r requirements.txt
+- pip install pypandoc
```

Comment:
Would it make sense to have a development requirements.txt so we don't need to manually add new packages?

Reply:
This is only for use on Travis since PyPi needs RST for documentation.